src/prompt/prompt.js

```diff
@@ -116,10 +116,10 @@ export async function promptForToolingConfigurations() {
     choices: choices.testChoices,
     message: 'Testing Framework:'
   }, {
-    name: 'database',
+    name: 'orm',
     type: 'list',
-    choices: choices.databaseChoices,
-    message: 'Database Framework:'
+    choices: choices.ormChoices,
+    message: 'ORM Tool:'
```

Comment:
Here can we use the full name instead of the acronym so as to not confuse any new developers?